internal/cmd/kafka/command_cluster_describe.go

```diff
@@ -101,8 +100,8 @@ func (c *clusterCommand) outputKafkaClusterDescription(cmd *cobra.Command, clust
 
 	if getTopicCount {
 		topicCount, err := c.getTopicCountForKafkaCluster(cluster)
-		if err != nil {
-			return err
+		if err != nil { // topicCount is 0 when err != nil, and will be omitted by `omitempty`
+			output.ErrPrintln("WARNING: The CLI was unable to retrieve Kafka topics for this cluster. The topic count will be omitted.")
```

Comment:
How about we log this to DEBUG? I don't think we need to print this to stdout.

Reply:
I like it; I'll make that change.

Reply:
Actually, INFO would make more sense since it's underneath WARN